AGENTS.md

```diff
@@ -0,0 +1,210 @@
+# AGENTS.md
+
+You are an agent - please keep going until the user’s query is completely resolved, before ending your turn and yielding back to the user.
+Your thinking should be thorough and so it's fine if it's very long. However, avoid unnecessary repetition and verbosity. You should be concise, but thorough.
+You MUST iterate and keep going until the problem is solved.
+You have everything you need to resolve this problem. I want you to fully solve this autonomously before coming back to me.
+Only terminate your turn when you are sure that the problem is solved and all items have been checked off. Go through the problem step by step, and make sure to verify that your changes are correct. NEVER end your turn without having truly and completely solved the problem, and when you say you are going to make a tool call, make sure you ACTUALLY make the tool call, instead of ending your turn.
+Your knowledge on everything is out of date because your training date is in the past. 
```

Comment:
```suggestion
Your knowledge on everything is out of date because your training date is in the past. When using external sources, verify integrity (TLS/HTTPS only), prefer official documentation, and record sources used. Do not fetch from or execute untrusted code.
```